tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -733,88 +797,14 @@ def gmres_krylov(A_mv: Callable, A_args: Sequence, n_kry: int,
   ConstType = Tuple[float, Callable, Sequence, jax.ShapedArray, int]
   GmresCarryType = Tuple[VarType, ConstType]
 
-  @jax.jit
-  def gmres_krylov_loop_condition(gmres_carry: GmresCarryType) -> bool:
-    """
-    This function dictates whether the main GMRES while loop will proceed.
-    It is equivalent to:
-      if k < n_kry and err > tol:
-        return True
-      else:
-        return False
-    where k, n_kry, err, and tol are unpacked from gmres_carry.
-
-    Args:
-      gmres_carry: The gmres_carry from gmres_krylov.
-    Returns:
-      (bool): Whether to continue iterating.
-    """
-    gmres_constants, gmres_variables = gmres_carry
-    tol = gmres_constants[0]
-    k = gmres_variables[0]
-    err = gmres_variables[4]
-    n_kry = gmres_constants[4]
-
```

Comment:
Did you mean to delete all of this?

There's a lot of code changes for what should be a simple change.

Reply:
The code has been shifted into the scope of the `precision` variable. Other than that it hasn't changed, but it looks kind of messy here ...